code/game/objects/items/devices/multitool.dm

```diff
@@ -169,3 +169,14 @@
 	icon = 'icons/obj/abductor.dmi'
 	icon_state = "multitool"
 	toolspeed = 0.1
+
+/obj/item/multitool/abductor/syndi
+	name = "multitool"
+	desc = "Used for pulsing wires to test which to cut. Not recommended by doctors."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "multitool"

```

Comment:
this will actually inherit, my bad.

Is there a particular reason you're pulling from abductor though? If not consider subtyping off multitool.